cooperator/views/subscription_request_view.xml

```diff
@@ -27,13 +27,13 @@
                 <field name="partner_id" />
                 <field name="source" />
                 <field name="state" />
-                <field name="validated" invisible="True" />
+                <field name="is_valid_iban" invisible="True" />
                 <button
                     type="object"
                     string="Validate"
```

Comment:
```suggestion
                    string="Validate IBAN"
```

Reply:
This one is to validate the subscription request and send an invoice to the cooperator candidate.